examples/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+services:
+    tusd:
+      image: tusproject/tusd:v1.9
+      command: -verbose -s3-bucket mybucket -s3-endpoint http://minio:9000
+      volumes:
+        - tusd:/data
+      environment:
+        - AWS_REGION=us-east-1
+        - AWS_URL=http://minio:9000
```

Comment:
tusd does not use the AWS_URL environment variable. Is it used by some other component?